handsontable/src/plugins/hiddenColumns/hiddenColumns.js

```diff
@@ -80,6 +81,49 @@ export const PLUGIN_PRIORITY = 310;
  * // to see your changes, re-render your Handsontable instance
  * hot.render();
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * const hotRef = useRef();
+ *
+ * ...
+ *
+ * <HotTable
+ *   ref={hotRef}
+ *   data={getData()}
+ *   hiddenColumns={{
+ *     copyPasteEnabled: true,
+ *     indicators: true,
+ *     columns: [1, 2, 5]
+ *   }}
+ * />
+ *
+ * // access the `HiddenColumns` plugin's instance
+ * const hiddenColumnsPlugin = hotRef.getPlugin('hiddenColumns');
```

Comment:
Same as above - you'll need to call the API methods from the `.current.hotInstance` property of the referenced component.

Reply:
Changed within: https://github.com/handsontable/handsontable/pull/9791/commits/8464a8efd8e87971f4d593025622e98b0e8b20ae